### PR TITLE
JBPM-4739 - Create a Process Instance and Task webservice

### PR DESCRIPTION
**Pull request by mrietveld**
Also meant for cherry-pick to 6.3.x.

### CODE REVIEW

File: drools-core/src/main/java/org/drools/core/command/runtime/process/SignalEventCommand.java [hunk 1]
```diff
@@ -107,8 +121,10 @@ public Void execute(Context context) {
     }
 
     public String toString() {
-        if (processInstanceId == -1) {
-            return "ksession.signalEvent(" + eventType + ", " + event + ");";
+        if (processInstanceId == -1 && correlationKey == null) {
+            return "ksession.signalEvent(" + eventType + ", " + event + ");"; 
+        } else if (correlationKey != null) {
+            return "ksession.signalEvent(" + correlationKey + ", " + eventType + ", " + event + ");";
```
Comment (mswiderski) [inline drools-core/src/main/java/org/drools/core/command/runtime/process/SignalEventCommand.java:135]: minor thing but would be better to align toString with actual method signature as it sort of simulates it - meaning processInstanceId/correlationKey to have as last parameter (signal by correlationKey seems not to be in place yet if ever planned to be there)

besides that looks good to me

Comment (mrietveld) [inline drools-core/src/main/java/org/drools/core/command/runtime/process/SignalEventCommand.java:135]: I agree, but that would also mean modifying the toString() method below -- and I try not to change anything that's not related to the change I want to make.

**Comment by triceo** [comment]
As far as I know, BPMSPL-166 is no longer approved for the release - please do not cherry-pick to 6.3.x.

**Comment by mrietveld** [comment]
Yup, just saw that today. Thanks @triceo!

(BTW, that is a _big_ mug of beer! ;D)

**Comment by triceo** [comment]
Nah, _no mug_ is too big. :-)

**Pull request merged**